site/ja/guide/function.ipynb

```diff
@@ -486,16 +485,61 @@
         "v"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "mxjNpA6Ua4_o"
+      },
+      "source": [
+        "<a id=\"debugging\"></a>\n",
+        "\n",
+        "## デバッグ\n",
+        "\n",
+        "`tf.function` や AutoGraph はコードを生成し、それを TensorFlow のグラフにトレースします。この機構は現在のところ、`pdb` のようなステップ・バイ・ステップのデバッガーをサポートしていません。ですが、`tf.config.experimental_run_functions_eagerly(True)` を呼び出すことで、一時的に Eager Execution を `tf.function` の中で有効にして、好きなデバッガーを用いることができます。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "i1HtHX6Ja4_o"
+      },
+      "outputs": [],
+      "source": [
+        "@tf.function\n",
+        "def f(x):\n",
+        "  if x > 0:\n",
+        "    # Try setting a breakpoint here!\n",
```

Comment:
Do we need to translate this into Japanese?

Reply:
IMO, it is so hard to maintain these comments in the code cell translated into Japanese. I guess, with the code sync tool `nb_code_sync.py`, this comment will be overwritten by the original one when the original English notebook was updated.

I translate these comments into Japanese just now, however, we need to discuss this issue in the Japanese translation community.